### ISSUE TITLE
Logs not showing up in repl when using Clojure CLI

### ISSUE THREAD

**Issue by noahmoss** [liquidz/vim-iced#359]
Hi @liquidz!

At [Metabase](https://github.com/metabase/metabase) we just switched from using Leiningen to using the Clojure CLI with deps.edn.

vim-iced is working pretty smoothly still, however I have one issue: logs are not showing up in the repl when I start it with `iced repl --without-cljs -A:dev`. Logs do work properly, however, when I start the repl with `clj -A:dev`, so it seems like might be a problem with the `iced` command? We are using `clojure.tools.logging` and log4j2, and printing to stdout still works fine, just not logging.

Do you have any idea what the problem might be? Thanks!

**Comment by liquidz** [comment]
@noahmoss Thanks for your reporting!

As far as I tried, it seems [cider-nrepl](https://github.com/clojure-emacs/cider-nrepl)'s `wrap-debug` middleware is the cause.
I'll dig into the problem later.

As a workaround, editing `/your-iced-directory/bin/iced` like follows will show logs in your REPL (but you cannot use debugging feature of course)
```diff
-BASE_MIDDLEWARES='cider.nrepl/wrap-classpath cider.nrepl/wrap-clojuredocs cider.nrepl/wrap-complete cider.nrepl/wrap-debug cider.nrepl/wrap-format cider.nrepl/wrap-info cider.nrepl/wrap-macroexpand cider.nrepl/wrap-ns cider.nrepl/wrap-out cider.nrepl/wrap-refresh cider.nrepl/wrap-spec cider.nrepl/wrap-test cider.nrepl/wrap-trace cider.nrepl/wrap-undef cider.nrepl/wrap-xref refactor-nrepl.middleware/wrap-refactor iced.nrepl/wrap-iced'
+BASE_MIDDLEWARES='cider.nrepl/wrap-classpath cider.nrepl/wrap-clojuredocs cider.nrepl/wrap-complete cider.nrepl/wrap-format cider.nrepl/wrap-info cider.nrepl/wrap-macroexpand cider.nrepl/wrap-ns cider.nrepl/wrap-out cider.nrepl/wrap-refresh cider.nrepl/wrap-spec cider.nrepl/wrap-test cider.nrepl/wrap-trace cider.nrepl/wrap-undef cider.nrepl/wrap-xref refactor-nrepl.middleware/wrap-refactor iced.nrepl/wrap-iced'
```

**Comment by noahmoss** [comment]
Awesome, thanks!

**Comment by liquidz** [comment]
@noahmoss Instead of cider-nrepl 0.26.0, 0.25.11 seems work well.
So I downgraded cider-nrepl to 0.25.11 for now, and cut a new release #360 
Could you try latest version?

I'll continue to dig into cider-nrepl's problem.

**Comment by noahmoss** [comment]
@liquidz Yes, downgrading seems to have fixed it, thanks!

**Comment by bootleq** [comment]
We should still consider upgrading cider-nrepl, recently I am suffering problem like https://github.com/clojure-emacs/cider-nrepl/issues/719, where re-throwed exception may block iced from reporting test error.

I am using aforementioned workaround editing `bin/iced` to specify cider-nrepl `0.27.3` currently.

**Comment by liquidz** [comment]
@bootleq Sorry to bother you.
Yes, we should.
I'll test the latest cider-nrepl this weekend.

**Comment by liquidz** [comment]
@bootleq
I just updated cider-nrepl to 0.27.4 in `dev` branch for now.

@noahmoss 
Currently, logs seem to be shown on REPL in my environment.
Could you try `dev` branch?

**Comment by noahmoss** [comment]
@liquidz Just tried the `dev` branch and logging seems to be working!

**Comment by liquidz** [comment]
@noahmoss Thanks for your confirmation!

**Comment by liquidz** [comment]
@noahmoss @bootleq Just released v3.7.3 #377
